packages/validation-manager/src/ValidationManager.ts

```diff
@@ -110,27 +148,40 @@ export class ValidationManager implements IValidationManager {
     }
   }
 
+  allOnes = '0x'.padEnd(66, 'f')
+
+  // a "flag" UserOperation that triggers "AA94" revert error (not even FailedOp)
```

Comment:
Can you explain the rationale? That we inject a reverting userOp to avoid simulating the execution, and explicitly mention that revert on the eolUserOp means success?

Reply:
the handleOps validate the userop before calling it.
Thus FailedOp(index=1, "AA94") means it is an invalid UserOp
is it better to have a revert inside validateUserOp ?

Reply:
Okay, I need more context to understand what is going on here.